code/modules/library/skill_learning/job_skillchips/research_director.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/skillchip/job/research_director
+	name = "R.D.S.P.L.X. skillchip"
+	desc = "Knowledge of how to solve the ancient conumdrum; what happens when an unstoppable force meets an immovable object."
+	auto_trait = TRAIT_ROD_SUPLEX
+	skill_name = "True Strength"
+	skill_description = "The knowledge and strength to resolve the most ancient conumdrum; what happens when an unstoppable force meets an immovable object."
+	skill_icon = "dumbbell"
+	activate_message = "<span class='notice'>You feel the urge to prove yourself in the upcoming Feats of Strength.</span>"
+	deactivate_message = "<span class='notice'>You no care about the upcoming Feats of Strength, and are no longer clear on what they were in the first place.</span>"
```

Comment:
Could you change this message to be a bit nerdier?
It's the Research Director, not an athlete.